<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Youtube video Downloader</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(16, 20, 24);
border:1px solid rgb(158, 158, 158);
border-radius: 10px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="url_entry">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color:rgb(16, 20, 24);
	border:2px solid rgb(158, 158, 158);
	border-radius:10px;
	color:rgb(158, 158, 158);
	font-size:30px;

}</string>
    </property>
    <property name="placeholderText">
     <string>Paste the URL here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paste_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>330</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: rgb(10, 115, 237);
	border: 10px;
	border-radius:  30px;
	color:rgb(255, 255, 255);
	
}

QPushButton:hover{
	background: rgb(6, 90, 188);
}
</string>
    </property>
    <property name="text">
     <string>Paste the URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download_btn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>330</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:rgb(232, 0, 0);
	border: 10px;
	border-radius:  30px;
	color:rgb(255, 255, 255);
	
}

QPushButton:hover{
	background:rgb(193, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>Download video</string>
    </property>
   </widget>
   <widget class="QLabel" name="download_ani">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>751</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border: 0px solid;
	color: rgb(0, 170, 62);
	font-family: Helvertica;
	font-size: 30px;
	font-weight: 10px;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="titleBar">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>1085</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	
	background-color: rgb(37, 46, 54);
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>0</y>
       <width>61</width>
       <height>60</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
		background-color: rgb(37, 46, 54);
		border: 10px;
		border-radius: 0px;
		
}

QPushButton:hover{
	background:rgb(244, 67, 54);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resources/close.png</normaloff>../resources/close.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="minimizeButton">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>0</y>
       <width>51</width>
       <height>60</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	
	background-color: rgb(37, 46, 54);
	border: 1px;
	border-radius: 0px;
}

QPushButton:hover{
	background:rgb(44, 55, 67);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resources/minimize.png</normaloff>../resources/minimize.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: 0px solid;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../resources/logo_title.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>25</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color: white;
	font-family: ADLaM Display;
	font-size: 15px;
	

}</string>
     </property>
     <property name="text">
      <string>Youtube video downloader by C.C</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border: 0 px solid;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../resources/main_logo_final.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>64</y>
      <width>1085</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Line{
	border: 0px;
	background-color:  rgb(158, 158, 158);
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
